src/Channel.ts

```diff
@@ -162,14 +187,24 @@ export default class Channel {
   }
 
   // give the channel fresh volume data and initialize from that data
-  public setFromVolumeData(bitsArray: Uint8Array, vx: number, vy: number, vz: number, ax: number, ay: number): void {
+  public setFromVolumeData(
+    bitsArray: Uint8Array,
+    vx: number,
+    vy: number,
+    vz: number,
+    ax: number,
+    ay: number,
+    rawMin = 0,
+    rawMax = 255
+  ): void {
     this.dims = [vx, vy, vz];
     this.volumeData = bitsArray;
     // TODO FIXME performance hit for shuffling the data and storing 2 versions of it (could do this in worker at least?)
     this.packToAtlas(vx, vy, vz, ax, ay);
     this.loaded = true;
+    // update from current histogram?
+    this.setRawDataRange(rawMin, rawMax);
```

Comment:
this is where I automatically remap the lookup table, as soon as a Channel receives new volume data.